config-connector/solutions/apps/helm/wordpress/charts/wordpress-gcp/templates/k8s-stateful-set.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  namespace: default
```

Comment:
Same here. Which namespace(s) do we want to use? "default"? Default? Or both?

Reply:
Current namespace. Fixed.